snmp/datadog_checks/snmp/data/profiles/aruba.yaml

```diff
@@ -1,92 +1,14 @@
-extends:
-  - _base.yaml
-  - _generic-if.yaml
-  - _generic-ospf.yaml
-
-device:
-  vendor: "aruba"
+# Profile for Aruba devices
+# Legacy profile that match all Aruba devices
+# Newer profiles are: aruba-switch.yaml, aruba-access-point.yaml, etc
 
 sysobjectid: 1.3.6.1.4.1.14823.*
 
-metrics:
-  ### Sensor
-
-  # The table of all supported fans in the controller.
-  # Not supported in Aruba 200/800 and 2400 controllers.
-  - MIB: WLSX-SYSTEMEXT-MIB
-    table:
-      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17
-      name: wlsxSysExtFanTable
-    symbols:
-      - # NOTE: active(1), inactive(2)
-        OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17.1.2
-        name: sysExtFanStatus
-    metric_tags:
-      - tag: fan_index
-        column:
-          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17.1.1
-          name: sysExtFanIndex
-
-  ### Power
-
-  # The table of Power supplies in the controller.
-  # Not supported in Aruba 200, 800 and 2400 controllers.
-  - MIB: WLSX-SYSTEMEXT-MIB
-    table:
-      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.18
-      name: wlsxSysExtPowerSupplyTable
-    symbols:
-      # NOTE: active(1), inactive(2)
-      - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.18.1.2
-        name: sysExtPowerSupplyStatus
-    metric_tags:
-      - tag: powersupply_index
-        column:
-          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.18.1.1
-          name: sysExtPowerSupplyIndex
-
-  ### Processor
-
-  - # The table of processors contained by the controller.
-    MIB: WLSX-SYSTEMEXT-MIB
-    table:
-      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.13
-      name: wlsxSysExtProcessorTable
-    symbols:
-      # The average, over the last minute, of the percentage of time that this processor was not idle.
-      - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3
-        name: sysExtProcessorLoad
-    metric_tags:
-      - tag: processor_index
-        column:
-          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.13.1.1
-          name: sysExtProcessorID
-
-  ### Memory
-
-  - # The memory status of the controller. All memory values measured in KB.
-    MIB: WLSX-SYSTEMEXT-MIB
-    table:
-      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1
-      name: wlsxSysExtMemoryTable
-    symbols:
-      - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1.2
-        name: sysExtMemorySize
-      - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1.3
-        name: sysExtMemoryUsed
-      - OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1.4
-        name: sysExtMemoryFree
-    metric_tags:
-      - tag: memory_index
-        column:
-          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.15.1.1
-          name: sysExtMemoryIndex
-
-
-  ### Interface
+device:
+  vendor: "aruba"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Can we move also `device.vendor: "aruba"` to the base aruba profile ?

Reply:
Yes, thought about that, but we can't for now, the current implementation is reading `device.vendor` only for the leaf profile.

(It's probably possible to support reading from a base profile, but not sure it's worth it since we are moving to `metadata.device.fields.vendor` implementation).